docs/concepts/projects.md

```diff
@@ -133,24 +132,36 @@ For more information on the mechanics of steps, [see below](#deployment-mechanic
+The build section of `prefect.yaml` is where any necessary side effects for running your deployments are built - the most common type of side effect produced here is a Docker image.  If we initialize with the docker recipe we are met with a series of prompts to provide required information such as image name and tag: 
 
 <div class="terminal">
 ```bash
 $ prefect project init --recipe docker
+>> image_name: < insert image name here >
+>> tag: < insert image tag here >
 ```
 </div>
 
+!!! tip "Use `--field` to avoid the interactive experience"
+    We recommend that you only initialize a recipe when you begin a project, and afterwards store your configuration files within version control; however, sometimes you may need to initialize programmatically and avoid the interactive prompts.  To do so, provide all required fields for your recipe using the `--field` flag:
+    <div class="terminal">
+    ```bash
+    $ prefect project init --recipe docker \
+        --field image_name=my-repo/my-image \
+        --field tag=my-tag
+    ```
+    </div>
+    
 ```yaml
 build:
 - prefect_docker.projects.steps.build_docker_image:
     requires: prefect-docker>=0.2.0
-    image_name: null
-    tag: null
+    image_name: my-repo/my-image
+    tag: my-tag
     dockerfile: auto
 ```
 
-Once we edit the `null` fields to their desired values, this step will automatically build a Docker image with the provided name and tag and push it to the repository referenced by the image name.  [As the documentation notes](https://prefecthq.github.io/prefect-docker/projects/steps/#prefect_docker.projects.steps.BuildDockerImageResult), this step produces a few fields that can optionally be used in future steps or within `deployment.yaml` as template values.  It is best practice to use `{{ image_name }}` within `deployment.yaml` (specificially the work pool's job variables section) so that you don't risk having your build step and deployment specification get out of sync with hardcoded values.  For a worked example, [check out the project tutorial](/tutorials/projects/#dockerized-deployment).
+Once we confirm that these fields are set to their desired values, this step will automatically build a Docker image with the provided name and tag and push it to the repository referenced by the image name.  [As the documentation notes](https://prefecthq.github.io/prefect-docker/projects/steps/#prefect_docker.projects.steps.BuildDockerImageResult), this step produces a few fields that can optionally be used in future steps or within `deployment.yaml` as template values.  It is best practice to use `{{ image_name }}` within `deployment.yaml` (specificially the work pool's job variables section) so that you don't risk having your build step and deployment specification get out of sync with hardcoded values.  For a worked example, [check out the project tutorial](/tutorials/projects/#dockerized-deployment).
```

Comment:
```suggestion
Once you've confirmed that these fields are set to their desired values, this step will automatically build a Docker image with the provided name and tag and push it to the repository referenced by the image name.  [As the documentation notes](https://prefecthq.github.io/prefect-docker/projects/steps/#prefect_docker.projects.steps.BuildDockerImageResult), this step produces a few fields that can optionally be used in future steps or within `deployment.yaml` as template values.  It is best practice to use `{{ image_name }}` within `deployment.yaml` (specificially the work pool's job variables section) so that you don't risk having your build step and deployment specification get out of sync with hardcoded values.  For a worked example, [check out the project tutorial](/tutorials/projects/#dockerized-deployment).
```